Misc/NEWS.d/next/Library/2020-01-06-22-39-23.bpo-39206.zKSTwb.rst

```diff
@@ -0,0 +1 @@
+Add encoding detecting to _find_module() in modulefinder
```

Comment:
maybe something like

```rst
Fix :mod:`modulefinder` to handle :pep:`263` encoding markers - by Firstname Lastname.
```

Reply:
Thanks for the suggestion, I will change it. Is there any guideline for creating this?

Reply:
I usually find whatever previous one I've done -- this is the closest I could find to guidance in the devguide: https://devguide.python.org/committing/#what-s-new-and-news-entries

Reply:
Thanks, I used your suggestion.